graylog2-web-interface/src/components/visualizations/QuickValuesVisualization.jsx

```diff
@@ -205,7 +210,12 @@ const QuickValuesVisualization = React.createClass({
     ];
 
     if (this.props.displayAddToSearchButton) {
-      columns.push(d => this._getAddToSearchButton(d.term));
+      if (this.props.field === 'timestamp') {
+        // convert unix timestamp to proper formatted value, so that add to search button works correctly
+        columns.push(d => this._getAddToSearchButton(moment.utc(Number(d.term)).format(DateTime.Formats.TIMESTAMP)));
```

Comment:
This also has problems with the user TZ, as the value from here is later converted into UTC (see gif below) and when pressing the search button we can't find any results. We should use, as in the other case `new DateTime(Number(d.term)).toString(DateTime.Formats.TIMESTAMP)` to convert the unix time in the user's selected time zone.

![time](https://user-images.githubusercontent.com/716185/34051293-4f14ea64-e1be-11e7-9fcc-9aea4954b35b.gif)

Reply:
By the way, as both date times should be formatted the same way, I would extract the conversion into a function for extra clarity.